.buildkite/e2e/nightly-main-matrix.yaml

```diff
@@ -28,7 +28,7 @@
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.25.3@sha256:f52781bc0d7a19fb6c405c2af83abfeb311f130707a0e219175677e366cc45d1
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.26.3@sha256:61b92f38dff6ccc29969e7aa154d34e38b89443af1a2c14e6cfbd2df6419c66f
     - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.27.1@sha256:c44686bf1f422942a21434e5b4070fc47f3c190305be2974f91444cd34909f1b
-    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.27.1@sha256:c44686bf1f422942a21434e5b4070fc47f3c190305be2974f91444cd34909f1b
+    - DEPLOYER_KIND_NODE_IMAGE: kindest/node:v1.28.0@sha256:b7a4cad12c197af3ba43202d3efe03246b3f0793f162afb40a33c923952d5b31
```

Comment:
Have you checked if this image which seems to be build for kind 0.20.0 works with kind 0.17.0 (which seems what we are currently using)

Reply:
Good point, thanks for the reminder. I have now tested kind 0.17 + k8s 1.28.0 images, installing Eck and a single node Elasticsearch cluster worked.

Tangential question: why did we have kindest/node:v1.27.1 listed twice? Is it ok to remove one of them?

Reply:
https://github.com/elastic/cloud-on-k8s/blob/6196b31dd06aa267fad3d0dc770dad57c526afbd/.buildkite/e2e/nightly-main-matrix.yaml#L30-L32

We use the latest kind version to test ipv6. The second occurence defines the ip family to ipv6 (default is ipv4).

Reply:
Thanks for the clarification! I guess we would still like to keep that, right? If so, should I add 1.28.0 separately (while also keeping 1.27.1 + ipv4 and 1.27.1 + ipv6)?

Reply:
👍  (https://github.com/elastic/cloud-on-k8s/pull/7129/commits/8d16149c05e637192176186ee5a0d7d38b527d13)

Should we also update deployer to use the 1.28 images by default (or should we wait until we see tests pass)?

https://github.com/elastic/cloud-on-k8s/blob/main/hack/deployer/config/plans.yml#L165-L186

Reply:
We can test almost anything right now by calling `buildkite test this -f ...`, see https://github.com/elastic/cloud-on-k8s/tree/main/.buildkite/e2e/pipeline-gen.